mappings/net/minecraft/data/server/recipe/ComplexRecipeJsonFactory.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS ha net/minecraft/data/server/recipe/ComplexRecipeJsonFactory
+	CLASS ha$1
+		METHOD a (Lcom/google/gson/JsonObject;)V
+			ARG 1 arg1
```

Comment:
These shouldn't be mapped. I wonder why they got mapped?

Reply:
Those are local vars. I heard about the local var. mapping are broken after I mapped those things. How to revert?

Reply:
Don't worry about it.